test/driver/.gitignore

```diff
@@ -0,0 +1,6 @@
+_build
```

Comment:
Not that it matters at all, but I believe that most of these can be handled in the top level `.gitignore` globally like: `**rebar.lock`, but if you want to keep a specific one: `!/path/to/this/rebar.lock`. It just seems easier later on to figure out why or why not something is tracked by git.

Reply:
Should be fixed now